examples/simple/asyncapi.yaml

```diff
@@ -0,0 +1,39 @@
+asyncapi: 2.0.0
+info:
+  title: Account Service
+  version: 1.0.0
+  description: This service is in charge of the process 
```

Comment:
Maybe you want to update title and description? It's not related to account processing anymore.